src/runtime_src/xdp/profile/device/ioctl_monitors/ioctl_add.cpp

```diff
@@ -16,6 +17,7 @@
  */
 
 #ifndef _WIN32
+#ifndef SKIP_IOCTL
```

Comment:
Isn't this the same as:
```
#if !defined(_WIN32) && !defined(SKIP_IOCTL)
...
#endif
```

Reply:
You are correct, and I have modified the code in the last commit to clean up all these locations with the single line.